test/test.js

```diff
@@ -66,6 +66,26 @@ test('Storage', async t => {
   await fs.stat(join(XDG_STATE_HOME, 'filecoin-station', 'logs', 'modules'))
 })
 
+test('Station', async t => {
+  await t.test('Saturn', async t => {
+    const XDG_STATE_HOME = join(tmpdir(), randomUUID())
```

Comment:
On Windows, we should be using `LOCALAPPDATA` environment variable instead of `XDG_STATE_HOME`.

Reply:
Nice.

Do we need to consider macOS now? I don't remember if there is an env var we can use or if we are expected to follow the convention and place our local state files into `$HOME/Library/Caches`.

We do a bit of work in Station Desktop to find the correct platform-specific directory, see here: https://github.com/filecoin-station/desktop/blob/e3db3b4d2c7bdef37cbf7a3cc31d8047933d5733/main/consts.js#L21-L47

Maybe it's time to introduce `ROOT_DIR` so we don't need to duplicate this code. Or maybe we can extract that Desktop helper function into a standalone npm package. 🤷🏻

Reply:
Good call: https://github.com/filecoin-station/core/pull/70